subprojects/logging/src/main/java/org/gradle/internal/logging/config/LoggingRouter.java

```diff
@@ -21,4 +21,5 @@
 
 public interface LoggingRouter extends LoggingSystem, LoggingOutputInternal {
     void configure(LogLevel logLevel);
+    void configureMaxWorkerCount(int maxWorkerCount);
```

Comment:
I'm unsure calling this configuration method `configureMaxWorkerCount`. Should we simply use `maxWorkerCount(int)` instead?